app/build.gradle

```diff
@@ -26,6 +26,13 @@ android {
 
 dependencies {
     api project(':base')
+
+    implementation 'androidx.test.espresso:espresso-contrib:3.3.0'
```

Comment:
Try to use: `androidTestImplementation`. App will include this dependency only for Android tests, not for default build.